source/aat-terms.ttl

```diff
@@ -0,0 +1,21 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix sdo: <http://schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix : <https://id.kb.se/vocab/> .
+@prefix aat: <http://vocab.getty.edu/aat/> .
+@base <https://id.kb.se/term/aat/> .
+
+
+<Page> a :GenreForm ;
+       skos:prefLabel "Page"@en, "Sida"@sv ;
+       :inScheme </term/aat> ;
+       :broader aat:300194222 .
```

Comment:
Or `closeMatch`? And I can see that `:inScheme </term/aat>` will be a problem when we don´t have `exaktMatch`.